src/originalBudgetTrackingApp/export/outputVendors/csv/csv.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  EnrichedTransaction, ExportTransactionsFunction, OutputVendor, OutputVendorName
+} from '@/originalBudgetTrackingApp/commonTypes';
+import { mergeTransactions, sortByDate } from '@/originalBudgetTrackingApp/transactions/transactions';
+import { TransactionInstallments } from '@brafdlog/israeli-bank-scrapers-core/lib/transactions';
+import { promises as fs } from 'fs';
+
+const stringify = require('csv-stringify/lib/sync');
+const parse = require('csv-parse/lib/sync');
+
+export function parseTransactions(csvText: string) {
+  return parse(csvText, {
+    columns: true,
+    cast: (value, context) => {
+      switch (context.column) {
```

Comment:
- [ ] You have about 4 levels of indentations here, my suggestion is to move the `cast` to another function called something like `parseValue(value, columnName)`.

Also, personally most of the time I prefer to use an `object` as mapping, instead of a `switch` statement. But I can't explain to you how because the code is complicated.

- [ ] So another suggestion is also moving the conversions functions (inside the `cases`) to a separated function. When you're moving code to function, the function name can be the title of the code, and the code will be more understandable.

After these two suggestions I will iterate this file again, OK?

Reply:
ok